articles/custom-scroll-bar/index.md

```diff
@@ -0,0 +1,264 @@
+   
+3. Be able to implement a customized scrollbar on your web page using CSS.
+   
+4. Understand how we can target more browser support of this specification.
+
+### The Browser Scrollbar CSS properties 
+CSS has support for various properties that allow us to have control over the way the application scrollbar will look.
+The following are the available selectors for scrollbar customization include:
+- The `::-webkit-scrollbar` will represent the entire scrollbar.
+
+- `::-webkit-scrollbar-button` is the buttons on the scrollbar. The arrows that point upwards and downwards.
+  
+- The `::-webkit-scrollbar-thumb` is a draggable handle for making scrolls.
+  
+- `::-webkit-scrollbar-track` is the track or the progress bar.
+
+- `::-webkit-scrollbar-track-piece` represents the track that will not be covered by the handle when scrolling.
+  
+- `::-webkit-scrollbar-corner` is the bottom corner of the scrollbar, this is where both horizontal and vertical scrollbars meet.
+  
+- `::-webkit-resizer` is the draggable resizing handle that will appear at the bottom corner of the elements.
+  
+These are common properties in a -webkit vendor prefix. There exist various jQuery plugins that will extend or polyfill this functionality to other legacy browsers. A common plugin used is [jScrollPane](http://jscrollpane.kelvinluck.com/).
+
+### Targeting more browser support
+To build scrollbar customized styles, it is a good practice to target more support of this feature.
+This is where we need to write a CSS code that targets support for both `-webkit-scrollbar` and CSS Scrollbars specifications from W3C.
+This is an example that uses the properties of a scrollbar namely `scrollbar-width`, `scrollbar-color`, `::-webkit-scrollbar`, `::-webkit-scrollbar-track`, `::webkit-scrollbar-thumb`:
+```CSS
+/* This will work on Firefox */
+* {
+  scrollbar-width: thin;
+  scrollbar-color: blue orange;
+}
+
+/* Targtes on Chrome, Edge, and Safari */
+*::-webkit-scrollbar {
+  width: 12px;
+}
+
+*::-webkit-scrollbar-track {
+  background: orange;
+}
+
+*::-webkit-scrollbar-thumb {
+  background-color: blue;
+  border-radius: 20px;
+  border: 3px solid orange;
+}
+```
+ 
+> A side note: WebKit browsers will ignore some of the rules that are not recognized and make a fallback to apply 
+> apply the `-webkit-scrollbar` rules. An example, Firefox browsers will ignore rules that they are not recognizing and instead
+> use the CSS Scrollbars specifications.
+
+### Building a web page and implementing a customized scrollbar
+This is the part where I will design a landing page that implements a customized landing page. Let us create a folder for our project named `landing-page`. Inside this folder, we will create another folder and name it images` that will contain an image for our showcase section. At the root of our `landing-page`, we have two files, the `index.html` and `styles.css`.
```

Comment:
Hello, can we rephrase this first part of the sentence?
It will flow better without the _This is the part where I..._